src/System.IO.FileSystem.Watcher.Polling/tests/project.json

```diff
@@ -1,12 +1,22 @@
 {
+  "name": "System.IO.FileSystem.Watcher.Polling.Tests",
   "dependencies": {
-    "Microsoft.NETCore" : "5.0.0",
-    "System.Console":  "4.0.0-*",
-    "System.Threading.Thread": "4.0.0-*",
-    "System.Diagnostics.TraceSource": "4.0.0-*",
-    "xunit": "2.0.0-*"
+    "Microsoft.NETCore.TestHost": "1.0.0-beta-23409",
+    "Microsoft.NETCore.Platforms": "1.0.1-beta-23409",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.1-beta-23409",
+    "src": {
+          "type": "build",
+          "version": "1.0.0"
+        },
+    "Microsoft.NETCore": "5.0.0",
+    "System.Console": "4.0.0-beta-*",
```

Comment:
@KrzysztofCwalina Once we have dotnet test (I saw it working today), this will get much simpler. I will fix this then. Opened #549 to track this.